src/v/kafka/server/handlers/topics/types.h

```diff
@@ -67,6 +67,19 @@ static constexpr std::string_view topic_property_remote_delete
   = "redpanda.remote.delete";
 static constexpr std::string_view topic_property_segment_ms = "segment.ms";
 
+// Server side schema id validation
+static constexpr std::string_view topic_property_record_key_schema_id_validation
```

Comment:
these should be arrays `std::array<string_view,2>  [ "redpanda.*", "confluent.*"`]`

Reply:
That's not how I've implemented it, but there's something equivalent.